hilbert.h

```diff
@@ -17,7 +17,7 @@ namespace signalsmith { namespace hilbert {
 template<typename Sample>
 struct HilbertIIRCoeffs {
 	static constexpr int order = 12;
-	std::array<std::complex<Sample>, order> coeffs{{
+	const std::array<std::complex<Sample>, order> coeffs{{
```

Comment:
Were these giving warnings in a particular compiler, without `const`?  Or was this for tidiness? 😄